### ISSUE TITLE
show_cpu_context error 

### ISSUE THREAD

**Issue by arkup** [alexhude/uEmu#19]
Hi, Nice plugin worked well with previous IDA but with latest `IDA 7.4` when I try to show the CPU context (ARM32) I get an error:

```
WARNING: The method "ida_kernwin.UI_Hooks::finish_populating_tform_popup" won't be called (it has been replaced with "ida_kernwin.UI_Hooks::finish_populating_widget_popup")
[uEmu]: Init plugin uEmu

  File "C:/Program Files/IDA74/plugins/uEmu.py", line 1949, in show_cpu_context
    self.cpuContextView.SetContent(self.unicornEngine.pc, self.unicornEngine.mu)
  File "C:/Program Files/IDA74/plugins/uEmu.py", line 628, in SetContent
    for i in range(lines):
TypeError: 'float' object cannot be interpreted as an integer
```

**Comment by sobakaliza** [comment]
Same problem here

Update: managed to fix it! IDA+Python is a disaster! Seriously, every time I choose to use a Python plugin, I see execution problems [facepalm]

Anyway: Go to 628 and change / to // like that:

`lines = int(reg_cnt // cols) if reg_cnt % cols == 0 else (reg_cnt // cols) + 1`

**Comment by alexhude** [comment]
Fixed in f82b33d
